content/en/docs/languages/java/sdk.md

```diff
@@ -594,11 +602,18 @@ public class SdkMeterProviderConfig {
                 MetricReaderConfig.periodicMetricReader(
                     MetricExporterConfig.otlpHttpMetricExporter(
                         "http://localhost:4318/v1/metrics")));
+    // Uncomment to optionally register metric reader with cardinality limits
+    // builder.registerMetricReader(
+    //     MetricReaderConfig.periodicMetricReader(
+    //         MetricExporterConfig.otlpHttpMetricExporter("http://localhost:4318/v1/metrics")),
+    //     unusedInstrumentType -> 100);
```

Comment:
(the unused prefix could look odd for some people)
```suggestion
    //     instrumentType -> 100);
```

Reply:
https://github.com/open-telemetry/opentelemetry-java-examples/pull/527